examples/programming-lang-detection/README.md

```diff
@@ -9,13 +9,14 @@ The script is able to take in a file name and detect the kind of programming lan
 offers >90% accuracy for languages which we currently support of the file which is provided.
 ## Usage ##
 
-Run:
+You will need to have Keras-MXNet. 
 
+Run:
 ```
-python2.7 test.py <filename>
+python test.py <filename>
 ```
 
-You can download a pre-trained model from [here](). This model was trained on an AWS p3.16 instance and it took under 30 minutes to train.
+You can download a pre-trained model from [here](https://s3-us-west-2.amazonaws.com/email-boy-images/LabelBotModelFiles/lang_detection_model.h5). This model was trained on an AWS p3.16 instance and it took under 30 minutes to train.
```

Comment:
who is the owner of `email-boy-images` bucket?

Reply:
If you plan to have this bucket public accessible, please make sure it's white listed and does not have any legal issue.